source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,106 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
+
+1. Log into your Asana account.
+2. From the Asana Board Menu, select **Export/Print > JSON** to create an archive file (which you'll use in Boards).
```

Comment:
I wasn't able to find this option in my Asana account. Where is it located?

Reply:
I had trouble finding it originally as well. I added this to the readme.
From the Asana Board Menu (dropdown next to board title), select Export / Print, and JSON

<img width="720" alt="Screen Shot 2022-02-01 at 10 22 22 AM" src="https://user-images.githubusercontent.com/12704875/152018237-b1abd569-d1bf-4495-a506-43fc60771873.png">